docs/Azure_IoT_Integration_Guide/Configure_integration_templates/Azure_integration_templates.md

```diff
@@ -56,7 +56,11 @@ A dialog box will appear asking you to specify the group that needs to be assign
 !!! note
     One template can be assigned to more than one group.
 
-Now the devices in the **group ‘anjay’** will report data to Azure according to the way defined in the **template ‘abc_test’**. Coiote DM sends data to Azure every time a device data model is changed, provided that you have set an Observation on the objects and resources in this data model. (Read about setting an Observation in [Azure IoT Hub](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Azure_IoT_Hub_integration/Set_an_Observation/) and [Azure IoT Central](/Azure_IoT_Central_integration/Set_an_Observation/).)
+Now the devices in the **group ‘anjay’** will report data to Azure according to the way defined in the **template ‘abc_test’**.
+
+### Integration templates in action
```

Comment:
@Passarinho4 @mkrawczyk-avs   Does it look correct to you? (the whole "Integration templates in action" subsection) 
If anything needs to be changed (including the title of the subsection:) or you think it should be placed elsewhere, please let me know.

Reply:
Maybe "How integration templates work" is better?
Otherwise than that, I'm ok with the changes.

Reply:
@mkrawczyk-avs good idea, changed to "How integration templates work"